src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements Drawable, AreaCalculable {
+    private double sideA;
+    private double sideB;
+    private double height;
+
+    public IsoscelesTrapezoid(double sideA, double sideB, double height, String color) {
+        super.setColor(color);
+        this.sideA = sideA;
+        this.sideB = sideB;
+        this.height = height;
+    }
+
+    @Override
+    public double area() {
```

Comment:
The method name 'area' is a noun, which might be confusing as method names typically are verbs or actions. Consider renaming it to 'calculateArea' to clearly indicate that it performs an action and follows the verb convention for method names.